main.py

```diff
@@ -2,6 +2,40 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genre_tuple = (
+        "Western",
+        "Action",
+        "Dramma",
+    )
+    actors_tuple = (
+        "George Klooney",
+        "Kianu Reaves",
+        "Scarlett Keegan",
+        "Will Smith",
+        "Jaden Smith",
+        "Scarlett Johansson",
+    )
```

Comment:
You could make a tuple of tuples of strings and use unpacking during the iteration to avoid calling the `.split()` method for string. E.g.:
```
 actors_tuple = (
        ("George", "Klooney"),
        ("Kianu", "Reaves"),
        ...
```
And on line 26:
```
for first_name, last_name in actors_tuple:
```